app/views/layouts/application.html.haml

```diff
@@ -7,6 +7,7 @@
 
     = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track': 'reload'
     = javascript_include_tag 'application', 'data-turbolinks-track': 'reload'
+    %link{:href => "https://cdn.jsdelivr.net/npm/tailwindcss/dist/tailwind.min.css", :rel => "stylesheet"}/
```

Comment:
Out of scope for this feature.
If you want to add Tailwind to this application, you need to do that in a separate chore.

Reply:
Well, they do use the tailwind css classes. I would argue that the setup is relevant. But I agree that it could be done in a separate PR